### PR TITLE
LPS-91495 Apply DROP_TABLE_IF_EXISTS when usable.

### PR DESCRIPTION
**Pull request by marianoalvarosaiz**

### CODE REVIEW

**Comment by marianoalvarosaiz** [comment]
ci:test:sf

**Comment by marianoalvarosaiz** [comment]
ci:test:relevant

**Comment by marianoalvarosaiz** [comment]
ci:test:upgrade

**Comment by marianoalvarosaiz** [comment]
ci:test:upgrade

**Comment by marianoalvarosaiz** [comment]
ci:test:relevant

File: portal-impl/src/com/liferay/portal/dao/db/BaseDB.java [hunk 1]
```diff
@@ -332,6 +332,10 @@ public void runSQL(Connection con, String[] sqls)
 					sql = sql.substring(0, sql.length() - 3);
 				}
 
+				if (sql.endsWith("\n/")) {
+					sql = sql.substring(0, sql.length() - 2);
```
Comment (crodriguezy) [inline portal-impl/src/com/liferay/portal/dao/db/BaseDB.java]: :(

**Pull request closed without merging**